extensions/goose/src/goose-cli.ts

```diff
@@ -85,17 +85,27 @@ export class GooseCLI implements Disposable {
 
   generateCommandLine(
     flowPath: string,
-    options: { path: string; env?: Record<string, string> },
+    options: { path: string; env?: Record<string, string>; params?: Record<string, string> },
   ): FlowGenerateCommandLineResult {
     if (!this.cli?.path) throw new Error('goose not installed');
+    const args = ['run', '--recipe', flowPath];
+    if (options.params) {
+      for (const key in options.params) {
+        args.push('--params', `${key}=${options.params[key]}`);
+      }
+    }
```

Comment:
suggestion
```
const args = [
  'run',
  '--recipe',
  flowPath,
  ...(options.params
    ? Object.entries(options.params).flatMap(([key, value]) => ['--params', `${key}=${value}`])
    : [])
];

```

basically iterate on entries (that gives you both key and value items) rather than key to then get the value using the key